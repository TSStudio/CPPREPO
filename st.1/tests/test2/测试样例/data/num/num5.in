19 23 6