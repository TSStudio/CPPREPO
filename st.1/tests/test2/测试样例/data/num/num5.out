19 23 42 65 107 172 